### ISSUE TITLE
Assistant Output Veracity Checkbox

### ISSUE THREAD

**Issue by King-Darius** [LAION-AI/Open-Assistant#2095]
We should add some way of indicating that synthetic output is false.  We really should be using such data to train the reward model I imagine?

**Comment by AbdBarho** [comment]
@King-Darius which context exactly? like when collecting data on the website?

**Comment by King-Darius** [comment]
> @King-Darius which context exactly? like when collecting data on the website?

Yes, when ranking the synthetic outputs.

**Comment by AbdBarho** [comment]
We have a checkbox after the ranking, is this what you mean?

![image](https://user-images.githubusercontent.com/24505302/226107915-8d59a0bf-f32f-4489-83a8-1452d251a197.png)

**Comment by King-Darius** [comment]
> We have a checkbox after the ranking, is this what you mean?
> 
> ![image](https://user-images.githubusercontent.com/24505302/226107915-8d59a0bf-f32f-4489-83a8-1452d251a197.png)

Nope, we need something like that for each of the replies.  The reason being that it can be used in the training to weed out hallucinations and the like and make it easier to rank (rather than waste data).
